data/json/npcs/isolated_road/isolated_road_cody_dialogue.json

```diff
@@ -392,5 +397,37 @@
       { "text": "Not really interested.", "topic": "TALK_DONE" },
       { "text": "I'll think about it.", "topic": "TALK_BLACKSMITH_SERVICES" }
     ]
+  },
+  {
+    "id": "TALK_BLACKSMITH_PROTOTYPE_NOT_FEELING_IT",
```

Comment:
@bombasticSlacks pinging you for this section, I'm not super familiar with them.

Reply:
This dialogue I think migrated files so this PR is duplicating it. 

The prototype not feeling it entry needs to go the way of the dodo